exercises/concept/simple-calculator/.docs/hints.md

```diff
@@ -1,2 +1,8 @@
 # Hints
 
+## General
+
+- Knowing about conditional statements such as `if/else` statements or `switch`
+is a prerequisite.
+- Having a list of permitted arguments can be useful and can make your code more
```

Comment:
Instead of "list", which might influence directly using an Array, or wondering where `list` is in Ruby, we can say "collection".

Reply:
Good catch @kotp